zebedee/client/client.go

```diff
@@ -39,25 +39,25 @@ var _ error = ErrInvalidZebedeeResponse{}
 
 // NewZebedeeClient creates a new Zebedee Client, set ZEBEDEE_REQUEST_TIMEOUT_SECOND
 // environment variable to modify default 5 second timeout
-func NewZebedeeClient(url string) ZebedeeClient {
+func NewZebedeeClient(url string) *ZebedeeClient {
 	timeout, err := strconv.Atoi(os.Getenv("ZEBEDEE_REQUEST_TIMEOUT_SECONDS"))
 	if timeout == 0 || err != nil {
 		timeout = 5
 	}
 
-	return ZebedeeClient{
+	return &ZebedeeClient{
 		zebedeeURL: url,
 		client:     &http.Client{Timeout: time.Duration(timeout) * time.Second},
 	}
 }
 
 // Get returns a response for the requested uri in zebedee
-func (c ZebedeeClient) Get(path string) ([]byte, error) {
+func (c *ZebedeeClient) Get(path string) ([]byte, error) {
 	return c.get(path)
 }
 
 // Healthcheck calls the healthcheck endpoint on the api and alerts the caller of any errors
-func (c *ZebedeeClient) Healthcheck() (string, error) {
+func (c ZebedeeClient) Healthcheck() (string, error) {
```

Comment:
Why not a pointer here but in all other healthchecks and other methods on this client?